doc/nrf/libraries/bootloader/bl_storage.rst

```diff
@@ -7,45 +7,23 @@ Bootloader storage
    :local:
    :depth: 2
 
-The bootloader storage library is used by the :ref:`bootloader` to read from and write to a protected data area that contains the provisioned bootloader data.
-The library can also be included into the application to allow reading the provisioned data.
+The bootloader storage library is used to read and write one-time programmable (OTP) data and a non-volatile counter.
+It may be used by both the |NSIB| (NSIB) and the application, but the application has read access only.
 
-The provisioned data is stored in the one-time programmable (OTP) region of UICR, if available (for example, on nRF9160 and the nRF5340 application core).
-In this case, it can only be read from applications that are in the secure domain.
-If OTP is not available, the data is stored in a regular flash page, and the bootloader blocks write access to it before booting the next image.
-In this case, some functions do not work when they are called from the application.
+The library has functions for either reading or writing, or in some cases both reading and writing, the following:
 
-The library stores the following data:
-
-* Image slot sizes
```

Comment:
Does it not store the sizes anymore or is this information encoded in PM ?

Reply:
I can't prove a negative and I don't know the history. But I see no API or allocation in OTP for image sizes.